### ISSUE TITLE
Run request - FWmadSD with interpolated EPP forcing

### ISSUE THREAD

**Issue by dan800** [UoL-Planetary-Modelling/cesm2.13_outofbox#1]
As before but with the following code mod copied to source mods directory.

In  src/chemistry/mozart/epp_ionization.F90 in subroutine create_input_obj (near line 405), add "force_time_interp=.true." optional argument to the call to in_obj%time_coord%initialize, like this:

    call in_obj%time_coord%initialize( path, force_time_interp=.true. )

Output to go to /resstore/b0243/Data

**Comment by dan800** [comment]
Suggest case name be f.e21.FWmadSD.f09_f09_mg17.2005-2020.002

**Comment by tashaay** [comment]
I've submitted a 5 day test run, the same as the previous run but with the codemod above. 

I've changed the DOUT_S_ROOT variable in env_run.xml to the b0243/Data/ folder, so hopefully any output should be directed straight into the relevant folder.

`<entry id="DOUT_S_ROOT" value="/resstore/b0243/Data/$CASE">`

If the case is successful, I'll upload the relevant info to a new repo.

**Comment by dan800** [comment]
That's great! Thanks.

**Comment by dan800** [comment]
> I've submitted a 5 day test run, the same as the previous run but with the codemod above.
> 
> I've changed the DOUT_S_ROOT variable in env_run.xml to the b0243/Data/ folder, so hopefully any output should be directed straight into the relevant folder.
> 
> `<entry id="DOUT_S_ROOT" value="/resstore/b0243/Data/$CASE">`
> 
> If the case is successful, I'll upload the relevant info to a new repo.

I don't see anything in the case directory and don't have read permissions for /nobackup/sestay//cesm2/cases/f.e21.FWmadSD.f09_f09_mg17.2005-2020.002/ so can't check on the run status. But I'm guessing there's an issue if no output has shown up?

**Comment by tashaay** [comment]
Yep I was an idiot and made the same error with the submission script again, I've resubmitted the job this morning, it's still queuing at the moment but hopefully won't be too long

**Comment by dan800** [comment]
> Yep I was an idiot and made the same error with the submission script again, I've resubmitted the job this morning, it's still queuing at the moment but hopefully won't be too long

Does the README reflect this problem with the submission script? If not, might be good to update it. 

edit: I just looked at the README and it's using planet in the script. I'll change it to using more than one feps nodes. It's 1˚ so I would imagine it would benefit from having more storage with 2 nodes. It would also be good to record the timings/cost info. for this configuration from the timings directory information - I've had a go, feel free to correct it.
